README.md

```diff
@@ -4,6 +4,12 @@
 
 Library for the [ArduinoLearningKitStarter (ALKS) board](https://github.com/RoboticsBrno/ArduinoLearningKitStarter/) by RoboticsBrno with definition of pins and initialization of peripheries.
 
+## Installation & code
+
+[Source code of library](https://github.com/RoboticsBrno/ArduinoLearningKitStarter-library/blob/master/src/ALKS.cpp)
```

Comment:
This must be in section `Available methods` because it is source code for the methods. Same as the [link for available pins source code](https://github.com/RoboticsBrno/ArduinoLearningKitStarter-library/pull/15/files#diff-04c6e90faac2675aa89e2176d2eec7d8R59) on the end of the README.md. 

Please hold the style of README.